src/indicators/drought/ecmwf_seasonal/processing.py

```diff
@@ -39,29 +40,44 @@ def get_ecmwf_forecast(
     years)
     """
     ecmwf_forecast = ecmwf_seasonal.EcmwfSeasonalForecast(**kwargs)
-    ds_ecmwf_forecast = ecmwf_forecast.read_processed_dataset(
-        country_iso3=country_iso3,
-        version=version,
-    )
+    if source_cds:
+        ds_ecmwf_forecast = ecmwf_forecast.read_processed_dataset(
+            country_iso3=country_iso3,
+            version=version,
+        )
+    else:
+        dataset_path = (
+            Path(os.environ["AA_DATA_DIR"])
+            / f"private/processed/{country_iso3}/ecmwf_test/"
```

Comment:
change ecmwf_test to ecmwf to make it a bit more ready to use :p

Reply:
Done!